spec/lib/rbac/filterer_spec.rb

```diff
@@ -1211,86 +1209,111 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
+            let!(:network_object) do
+              return network_manager if network_model == "ManageIQ::Providers::NetworkManager"
+              FactoryGirl.create(network_model.underscore, :ext_management_system => network_manager)
             end
-          end
 
-          it 'lists network manager according to belongsto filter' do
-            User.with_user(user) do
-              results = described_class.search(:class => ManageIQ::Providers::NetworkManager).first
-              expect(results).to match_array([network_manager])
+            let!(:network_object_with_different_network_manager) do
+              return network_manager_1 if network_model == "ManageIQ::Providers::NetworkManager"
+              FactoryGirl.create(network_model.underscore,  :ext_management_system => network_manager_1)
+            end
+
+            context 'when records match belognsto filter' do
+              it "lists records of #{network_model} manager according to belongsto filter" do
+                User.with_user(user) do
+                  results = described_class.search(:class => network_model).first
+                  expect(results).to match_array([network_object])
+                  expect(results.first.ext_management_system).to eq(network_manager)
+                end
+              end
+            end
+
+            context 'when records don\'t match belognsto filter' do
+              before do
+                group.entitlement = Entitlement.new
+                group.entitlement.set_managed_filters([])
+                group.entitlement.set_belongsto_filters(["/belongsto/ExtManagementSystem|XXXX"])
+                group.save!
+              end
+
+              it "lists no records of #{network_model}" do
+                User.with_user(user) do
+                  results = described_class.search(:class => network_model).first
+                  expect(results).to be_empty
+                end
+              end
             end
           end
         end
+      end
+
+      context 'network manager with/without tagging' do
+        let!(:cloud_network)     { FactoryGirl.create(:cloud_network, :ext_management_system => network_manager) }
+        let!(:cloud_network_1)   { FactoryGirl.create(:cloud_network, :ext_management_system => network_manager_1) }
 
-        context 'when records don\'t match belognsto filter' do
+        context 'network manager is tagged' do
           before do
             group.entitlement = Entitlement.new
-            group.entitlement.set_managed_filters([])
-            group.entitlement.set_belongsto_filters(["/belongsto/ExtManagementSystem|XXXX"])
+            group.entitlement.set_managed_filters([['/managed/environment/prod']])
+            group.entitlement.set_belongsto_filters([])
             group.save!
+
+            network_manager.tag_with('/managed/environment/prod', :ns => '*')
           end
 
-          it 'lists no cloud networks' do
+          it 'doesn\'t list cloud networks' do
```

Comment:
I think most specs use `"` to avoid these issues.

Reply:
👍  changed to ", in the separated commit, thanks

Reply:
It looks like this is another one.  The last commit fixes a different escaped single quote.

Reply:
yes I missed one context